task/buildah/0.1/buildah.yaml

```diff
@@ -331,7 +331,8 @@ spec:
     workingDir: $(workspaces.source.path)
 
   volumes:
-  - emptyDir: {}
+  - emptyDir:
+      medium: Memory
```

Comment:
Any concerns with this approach when performing a large build?

Reply:
good question @lcarva .... we dove into this a bit during the Aug 29 arch call.  The answer is "no" because
- the build team already has had to establish choices wrt the memory limits for the build container (stemming from hive's onboarding); per what you see in https://github.com/redhat-appstudio/build-definitions/tree/main/task we already have 4gb, 6gb, 8gb, 10gb choices, where more can be added
- the sandbox / RHTAP use of the cluster autoscaler should allocate more nodes on demand based on requests for more memory; 
- memory has been our least constrained resource to date
- memory is cheaper than storage wrt AWS
- the percentage delta per build from our stress runs was reasonable, especially when compared to the performance/scaling improvment